templates/404.html

```diff
@@ -46,27 +48,32 @@
 
         <main id="maincontent">
             <div class="generic-intro">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-12">
-                            <h1 class="tna-heading-xl generic-intro__title">
-                                Page not found
-                            </h1>
-                        </div>
-                    </div>
-                </div>
-            </div>
+                <div class="tna-container tna-container--centred">
+                    <div class="tna-column tna-column--full tna-column--width-1-2 tna-column--full-small tna-column--full-tiny tna-!--margin-vertical-l">
```

Comment:
Is there meant to be both a `tna-column--full` and a `tna-column--width-1-2`?

Reply:
Well spotted 👍 Now fixed